app/controllers/quotes_controller.rb

```diff
@@ -60,11 +60,17 @@ def create
 
   def show
     @quote = Quote.find(params[:id])
-    @customer_request = @quote.customer_request
+    @request = @quote.customer_request
     @company = @quote.company
     @view_date = @quote.view_date
-    if current_customer && !@view_date 
-      @quote.update(customer_viewed: true, view_date: Time.now) 
+    @quote.update(customer_viewed: true, view_date: Time.now) if current_customer && !@view_date
```

Comment:
I would place this line inside the if statement since we wouldn't want the quote to be updated if the user doesn't have permission to view it

Reply:
@melissacgonzalez @hellobrittany